index.html

```diff
@@ -0,0 +1,202 @@
+    </section>
+
+
+    <section class="multipost">
+      <img src="img/multipost.svg" class="postsvg">
+      <h2>Multi-Post<br>
+        Stories</h2><br>
+      <p class="inline-paragraph" id="canopy">CANOPY</p>
+      <img src="img/dot.svg">
+      <p class="inline-paragraph">Back End Dev</p>
+      <img src="img/dot.svg">
+      <p class="inline-paragraph">2023</p>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul>
+        <li class="my-button">HTML</li>
+        <li class="my-button">CSS</li>
+        <li class="my-button js">Javascript</li><br><br>
+      </ul>
+
+
+      <button type="button" class="see-project">See Project</button>
+    </section>
+
+
+    <section>
+      <img src="img/tonic2.svg" class="postsvg">
+      <h2>Tonic</h2><br>
+      <p class="inline-paragraph" id="canopy">CANOPY</p>
+      <img src="img/dot.svg">
+      <p class="inline-paragraph">Back End Dev</p>
+      <img src="img/dot.svg">
+      <p class="inline-paragraph">2023</p>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul>
+        <li class="my-button">HTML</li>
+        <li class="my-button">CSS</li>
+        <li class="my-button js">Javascript</li><br><br>
+      </ul>
+      <button type="button" class="see-project">See Project</button>
+    </section>
+
+
+    <section>
+      <img src="img/multi2.svg" class="postsvg">
+      <h2>Multi-Post<br>
+        Stories</h2><br>
+      <p class="inline-paragraph" id="canopy">CANOPY</p>
+      <img src="img/dot.svg">
+      <p class="inline-paragraph">Back End Dev</p>
+      <img src="img/dot.svg">
+      <p class="inline-paragraph">2023</p>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul>
+        <li class="my-button">HTML</li>
+        <li class="my-button">CSS</li>
+        <li class="my-button js">Javascript</li><br><br>
+      </ul>
+      <button type="button" class="see-project">See Project</button>
+    </section>
+
+
+    <section class="aboutmyself">
+      <h2>About
+        Myself</h2>
+      <p> Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my
+        work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+      </p>
+      <img src="img/LET’S CONNECT.svg" alt="humburger"><br><br>
+    </section>
+
+
+    <section>
+      <ul class="social-icons">
+        <li>
+          <img src="img/twitter icon.svg">
+        </li>
+        <li>
+          <img src="img/Linkedin icon.svg">
+        </li>
+        <li>
+          <img src="img/Mobile icon.svg">
+        </li>
+        <li>
+          <img src="img/git icon.svg">
+        </li>
+        <li>
+          <img src="img/peace icon.svg">
+        </li><br>
+      </ul>
+      <button type="button" class="getmy">Get my resume</button>
```

Comment:
- [ ] According to your previouse reviewer kindly Please make sure you added the hover and active properties to your button. Kindly check the Figma design,The button (e.g., "Get my resume,See project") is a link `a` or `button` tag, and follow the button styling: Active, Hover and Pressed (see design.
- [ ] ![Screenshot 2023-06-04 000218](https://github.com/Edgahkipkemoi/mobile/assets/92458236/e909995b-4946-403c-8285-ab721d671e93)
- [ ] see screenrecorf  

https://github.com/Edgahkipkemoi/mobile/assets/92458236/87f3ff1c-087a-4849-b717-a889596b1628